README.md

```diff
@@ -34,89 +34,108 @@ Loon is a repository that is meant to be deployed as a standalone docker contain
 
 ## Use And Deployment
 
-All of the necessary applications are shipped in a single docker container. In the repository, you will see two docker directories: `docker` and `docker-local`. The main deployment will use the `docker` directory. The `docker-local` directory is a separate local version of Loon which we will discuss shortly.
+All of the necessary applications are shipped in a single docker container. We use a python script called "build.py" to initiate and simplify the process.
+
+**Note that docker and Python 3+ must be installed and running in order to use the "build.py" script**
+
+We start by making a configuration file called "config.json". Below is a template:
+
+```json
+{
+  "generalSettings": {
+    "useHttp": false,
+    "environment": "production",
+    "baseUrl": "localhost"
+  },
+  "mySqlSettings": {
+    "databaseName": "loon",
+    "databaseUser": "user",
+    "databasePassword": "user_pass",
+    "databaseRootPassword": "root_pass",
+    "sourceVolumeLocation": "/Users/bbollen23/loonar-data/mysql-data"
```

Comment:
Volumes, in general, are for datasets which need to be persistent across restarts of an image. MySQL data needs to be persistent because when the user interacts with the UI, the MySQL database is updated.

But there is then two differences between different persistent volumes: specified path volumes and named volumes. A specified path would be something like your local data or minio data -- a volume that you want to be persistent AND you need to know where it is on your computer because you might want to interact with it outside of the container (i.e. local data would need to be a path specified volume because the user places data there).

For named volumes, these are volumes which need to be persistent but you don't need to really interact with them outside the ocntainer, like the MySQL volume. This has been changed to a name volume. When we use named volumes, the user doesn't need to specify any information for it -- it will automatically generate the volume for them in their docker volumes folder.